$MirOS: ports/mailnews/pine/patches/patch-pico_file_c,v 1.1.7.1 2005/03/18 15:49:48 tg Exp $
--- pico/file.c.orig	Fri Jun 11 21:49:26 2004
+++ pico/file.c	Wed May  4 23:28:08 2005
@@ -475,7 +475,7 @@ int     rename;         /* don't rename 
 	      if((s = ffgetline(line, NLINE, &charsread, 1)) == FIOEOF){
 		  curbp->b_flag &= ~(BFTEMP|BFCHG);
 		  gotobob(FALSE, 1);
-		  sprintf(line,"Read %d line%s",
+		  sprintf(line,"Read %ld line%s",
 			  nline, (nline > 1) ? "s" : "");
 		  emlwrite(line, NULL);
 		  break;
@@ -893,7 +893,7 @@ char    fname[];
 	      else
 		forwchar(FALSE, 1);
 
-	      sprintf(line,"Inserted %d line%s", nline, (nline>1) ? "s" : "");
+	      sprintf(line,"Inserted %ld line%s", nline, (nline>1) ? "s" : "");
 	      emlwrite(line, NULL);
 	      break;
 	  }
